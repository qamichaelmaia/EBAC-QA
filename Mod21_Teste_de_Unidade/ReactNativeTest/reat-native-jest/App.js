import { StyleSheet, Text, View } from 'react-native';

export default function App() {
  return (
    <View style={styles.container}>
      <Text>Meu primeiro App React Native na EBAC</Text>
    </View>
  );
}

const styles = StyleSheet.create({
  container: {
    flex: 1,
    backgroundColor: '#fff',
    alignItems: 'center',
    justifyContent: 'center',
  },
});
