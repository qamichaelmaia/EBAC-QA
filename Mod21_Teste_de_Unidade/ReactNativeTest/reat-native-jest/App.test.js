import { expect } from "@jest/globals";
import React from "react";
import renderer from 'react-test-renderer'
import App from "./App";

test('tela deve ser renderizada corretamente', () => {
    const app = renderer.create(<App/>).toJSON()
    expect(app).toMatchSnapshot()
});