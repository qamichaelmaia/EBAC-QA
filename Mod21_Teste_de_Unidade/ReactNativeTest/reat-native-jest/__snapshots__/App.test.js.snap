// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`tela deve ser renderizada corretamente 1`] = `
<View
  style={
    {
      "alignItems": "center",
      "backgroundColor": "#fff",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <Text>
    Meu primeiro App React Native na EBAC
  </Text>
</View>
`;
